<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_ejecutiv">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE ANALISTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="campaa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMPAÃ‘A</string>
    </property>
   </widget>
   <widget class="QLabel" name="activity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTIVIDAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="evalua">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ANALISTA EVAUDAOR</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="evaluation_date">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="evaluation_date_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FECHA EVALUACION</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="activity_2"/>
     </item>
     <item>
      <widget class="QComboBox" name="analista_calidad"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="campania">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>389</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="analyst_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_form">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEVALUAR_ANALISTA">
    <property name="title">
     <string>EVALUAR ANALISTA</string>
    </property>
   </widget>
   <addaction name="menuEVALUAR_ANALISTA"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
